mapping_project/lib/saved_locs.dart

```diff
@@ -1,159 +1,130 @@
 import 'package:flutter/material.dart';
+import 'custom_app_bar.dart';
 
-//All it needs for input is a List<string>, 
+//All it needs for input is a List<string>,
 //function returnList() returns updatedlist upon execution
 //could add functionality for reading/writing to txt file?
 //Lines 7 to 35 are purely for debugging
 
-void main() => runApp(MyApp());
+class SavedLocs extends StatefulWidget {
+  //Creates Stateful SavedLocs, allows for adding and removing locations
 
-class MyApp extends StatelessWidget {
-  // This widget is the root of your application.
   @override
-  Widget build(BuildContext context) {
-    return MaterialApp(
-      // title: 'Flutter Demo',
-      theme: ThemeData(
-        // This is the theme of your application.
-        //
-        // Try running your application with "flutter run". You'll see the
-        // application has a blue toolbar. Then, without quitting the app, try
-        // changing the primarySwatch below to Colors.green and then invoke
-        // "hot reload" (press "r" in the console where you ran "flutter run",
-        // or simply save your changes to "hot reload" in a Flutter IDE).
-        // Notice that the counter didn't reset back to zero; the application
-        // is not restarted.
-        primarySwatch: Colors.blueGrey,
-      ),
-      //home: MyHomePage(title: 'Flutter Demo Home Page'),
-      home: SavedLocs(["Room 101", "Room B6", "Room P5", "Computer Gallery"]),
-    );
-  }
+  _SavedLocsState createState() => _SavedLocsState();
 }
 
+class _SavedLocsState extends State<SavedLocs> {
+  //Actual savedlocs ofc
 
-
-
-
-class SavedLocs extends StatefulWidget {//Creates Stateful SavedLocs, allows for adding and removing locations
-final List<String> items;
-SavedLocs(this.items);
-
-  
-
-@override
-_SavedLocsState createState() => _SavedLocsState(items);
- }
-
-
-class _SavedLocsState extends State<SavedLocs> {//Actual savedlocs ofc
-
-
-
-final List<String> userLocs;
-
-_SavedLocsState(this.userLocs);
+  var userLocs = ["Room 1", "Room 2", "Room 3"];
```

Comment:
perhaps implement a getter from a file of the stored locations?

Reply:
Ah yes, missed that, will do later

Reply:
James is gonna do that actually - shall we merge and then he can edit it in?